plugins/media/wowza/lib/model/WowzaMediaServerNode.php

```diff
@@ -134,8 +134,8 @@ public function getApplicationPrefix($mediaServerConfig)
 
 		$appPrefix = $this->getValueByField($mediaServerConfig, 'appPrefix', $appPrefix);
 		
-		if($this->getAppPrefix())
-			$appPrefix = $this->getApplicationPrefix();
+		if(is_null($this->getAppPrefix()))
+			$appPrefix = $this->getAppPrefix();
```

Comment:
It seems that you will do this only in the case it is null - i think you want the opposite